examples/todo-list/nextjs-todo-list/supabase/config.toml

```diff
@@ -34,15 +34,14 @@ smtp_port = 54325
 pop3_port = 54326
 
 [storage]
-# The maximum file size allowed (e.g. "5MB", "500KB").
-file_size_limit = "50MiB"
+enabled = false
 
 [auth]
 # The base URL of your website. Used as an allow-list for redirects and for constructing URLs used
 # in emails.
 site_url = "http://localhost:3000"
 # A list of *exact* URLs that auth providers are permitted to redirect to post authentication.
-additional_redirect_urls = ["https://localhost:3000"]
+additional_redirect_urls = ["https://localhost:3000", "https://*.vercel.app/", "https:/*.vercel.app/**"]
```

Comment:
Hum, that might indeed be a security issue 🤔 

I did this because after a vercel deployment the local vs remote (created project) have a difference here. We automatically set the `additional_redirect_urls` based on the vercel deployed url. Vercel urls are like:

```
https://<project-and-branch-id>-<user-vercel-org>.vercel.app/
```

So maybe a more "production valid" example would be to add those lines templated and commented out. So when the user `link` the project he can see the diff ?

Reply:
I've reworked this section to retain the commented URLs instead of relying on Vercel globals matching. Additionally, I added a note in the `README.md` explaining that these will need to be merged with local values at `link` time.

That said, it's a fairly implicit flow. My concern is that while this setup works for basic cases, we’ll likely need examples for more "advanced usage." Specifically, examples that demonstrate how users can "glue things together" between their domain providers and their Supabase deployment. For instance, I envision something that explains how users can:

1. Deploy a domain/frontend on an arbitrary provider (e.g., using Vercel as an example).  
2. Extract values from this deployment.  
3. Replace or configure values in their Supabase environment.  
4. Initiate the preview deployment.  

Maybe something like a `use-actions` exemples or something like that WDYT?

Reply:
Yea, it sounds nice. Let's collect all these advanced use cases so we can plan the work on them after completing the basic examples.